packages/backend/src/index.ts

```diff
@@ -13,6 +14,11 @@ main().catch((e) => {
 
 async function main() {
   const cli = getCliParameters()
+  const env =
+    process.env.DEPLOYMENT_ENV ??
+    (process.env.NODE_ENV === 'production' ? 'production' : 'local')
+  console.log('Loading config for:', env)
+  const config = getConfig(env)
```

Comment:
Maybe a task to rethink what things are required in the config now needs to be added

Reply:
Added a task in linear